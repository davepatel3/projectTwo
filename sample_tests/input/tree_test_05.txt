int
allocate
42
0
allocate
47
0
set_parent
5
93861427785331
ref_node
2
allocate
18
0
allocate
-51
1
allocate
-76
3
allocate
-68
3
ref_node
1
allocate
-96
1
set_parent
3
4
set_parent
6
7
allocate
24
4
allocate
70
4
set_parent
7
0
allocate
96
6
allocate
-81
2
remove
5
remove
3
allocate
-73
4
allocate
99
8
remove
3
ref_node
5
allocate
-33
5
remove
5
remove
7
remove
11
set_parent
4
2
allocate
-70
9
allocate
62
4
ref_node
6
remove
3
set_parent
1
8
allocate
-51
9
set_parent
140182114764608
15
set_parent
10
5
allocate
-14
4
set_parent
6
9
set_parent
6
0
allocate
-43
0
ref_node
11
quit
