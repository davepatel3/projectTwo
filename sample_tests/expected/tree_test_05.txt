allocate
1
allocate
2
set_parent
Invalid handle!
ref_node
47
parent: 0
children: 
allocate
3
allocate
4
allocate
5
allocate
6
ref_node
42
parent: 0
children: 4 
allocate
7
set_parent
set_parent
allocate
8
allocate
9
set_parent
allocate
10
allocate
11
remove
remove
allocate
5
allocate
3
remove
ref_node
-73
parent: 4
children: 
allocate
3
remove
remove
remove
set_parent
allocate
3
allocate
5
ref_node
Node is recycled!
remove
set_parent
allocate
10
set_parent
Invalid handle!
set_parent
allocate
6
set_parent
set_parent
allocate
7
ref_node
Node is recycled!
quit
